### PR TITLE
Added linter configuration

### PR DESCRIPTION
**Pull request by Ugboaja-Uchechi**
Added linter styling to help debug and make the code neater

### CODE REVIEW

**Comment by Ugboaja-Uchechi** [comment]
Added linter styling to help debug, make the code neater and easy to understand

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Ugboaja-Uchechi,** 

### **To Highlight**
- The linters are passing.
- The Commit are good.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Ugboaja-Uchechi** [comment]
Added test.md file as required by my code reviewer.

File: test.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules/
```
Comment (AshakaE) [inline test.md:1]: As requested by the previous reviewer
- [ ] Please make sure this file is properly ignored by the `git` before making a `git push`

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Ugboaja-Uchechi  🙋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 # Required Changes ♻️ 🚧 
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test.md (hunk 1 above)
Comment (Ugboaja-Uchechi) [reply test.md]: If I have a .gitignore file consisting of the same information as the test.md file, do I still need the test.md file.

Comment (Ugboaja-Uchechi) [reply test.md]: I am not sure about the changes I'm to make.
1.  Is the test.md meant to be in the .gitignore file together with node_modules/
                              OR
2.  Is the test.md file meant to be a seperate file on its own

**Review by mke2111 — APPROVED** [review]
### Hi @Ugboaja-Uchechi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SpaYco — COMMENTED** [review]
Hello 👋,

_This is an additional review as a result of the second opinion request._

As this feature is still new, there are cases where a request wouldn't be noticed, sadly, yours was one of them, I just noticed this request now, and wanted to apologize for the delay

Thankfully I see that the problem got resolved and you continued with your projects!

Good luck with future projects and keep rocking 🚀!

**Pull request merged**